src/botPage/view/blockly/index.js

```diff
@@ -239,12 +237,16 @@ export const load = (blockStr, dropEvent = {}) => {
         const blockType = block.getAttribute('type');
 
         if (!Object.keys(Blockly.Blocks).includes(blockType)) {
-            const error = new TrackJSError(
-                'FileLoad',
-                translate('XML file contains unsupported elements. Please check or modify file.')
+            globalObserver.emit(
+                'Error',
+                createError(
+                    'InvalidBlockInXML',
+                    translate(
+                        'The file you’re trying to open contains unsupported elements in the following block: {$0} Please check your file and try again.',
+                        [block.getAttribute('id')]
+                    )
+                )
             );
-            globalObserver.emit('Error', error);
-            throw error;
```

Comment:
if we don't have the `throw error` first we have error message then `block message successfully loaded` so we need to break forEach function when we have a error,

Reply:
refactor this part. add find method to find at list 1 error. if err exist -> show pop up error.